src/helpers/eventsInterfaces/IOnSignEvent.ts

```diff
@@ -0,0 +1,6 @@
+export default interface IOnSignEvent {
+    type: 'onSign';
```

Comment:
please implement an ISimpleSignerEvent which has a type: string and a message: any and make all the other events extend from this one.